src/Open3D/GUI/Materials/metal.mat

```diff
@@ -1,6 +1,7 @@
 material {
     name : Metal,
     shadingModel : lit,
+    culling : none,
```

Comment:
Please, don't switch culling like this. If you need pretty rendering of back oriented faces, use `doubleSided`.

Reply:
Fixed